src/ngraph/runtime/generic_cpu/gcpu_executable.hpp

```diff
@@ -441,11 +437,9 @@ class ngraph::runtime::gcpu::GCPUExecutable : public Executable
                                        broadcast_axes);
             break;
         }
-        case OP_TYPEID::BroadcastDistributed: {
-#ifdef NGRAPH_DISTRIBUTED_ENABLE
-            Distributed dist;
-            int Rank_ID;
-            Rank_ID = dist.get_rank();
+        case OP_TYPEID::BroadcastDistributed:
+        {
+            int rank_ID = get_distributed_interface()->get_rank();
```

Comment:
`Rank_ID` is called below. I think the var names need to be consistent.

Reply:
Got it.